packages/react-components/react-overflow/src/useOverflowMenu.ts

```diff
@@ -2,32 +2,27 @@ import * as React from 'react';
 import { useId, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
 import { useOverflowContext } from './overflowContext';
 import { useOverflowCount } from './useOverflowCount';
-import { DATA_OVERFLOW_MENU } from './constants';
 
 export function useOverflowMenu<TElement extends HTMLElement>(id?: string) {
   const elementId = useId('overflow-menu', id);
   const overflowCount = useOverflowCount();
-  const registerItem = useOverflowContext(v => v.registerItem);
+  const registerOverflowMenu = useOverflowContext(v => v.registerOverflowMenu);
   const updateOverflow = useOverflowContext(v => v.updateOverflow);
   const ref = React.useRef<TElement>(null);
+  const offsetRef = React.useRef(0);
   const isOverflowing = overflowCount > 0;
 
   useIsomorphicLayoutEffect(() => {
     const element = ref.current;
-    if (element) {
-      const deregisterItem = registerItem({
-        element,
-        id: elementId,
-        priority: Infinity,
-      });
-      element.setAttribute(DATA_OVERFLOW_MENU, '');
-
-      return () => {
-        deregisterItem();
-        element?.removeAttribute(DATA_OVERFLOW_MENU);
-      };
+    if (!element || !isOverflowing) {
+      offsetRef.current = 0;
+      return;
     }
-  }, [registerItem, isOverflowing, elementId]);
+
+    offsetRef.current = 1;
```

Comment:
What is `offsetRef`? I don't see any usage/assignments outside of the effect.

Reply:
it's cruft :)